src/platforms/common/configuration/releases.mdx

```diff
@@ -53,6 +53,17 @@ Releases are global per organization; prefix them with something project-specifi
 
 </Note>
 
+The behavior of a few features depends on whether a project is using semantic or time-based versioning.
+
+- Regression detection
+- `release:latest`
+
+We automatically detect whether a project is using semantic or time-based versioning based on:
+
+- If ≤ 2 releases total: we look at most recent release.
+- If 3-10 releases total: if any of the most recent 3 releases is semver, project is semver.
+- If 10+ releases: if any of the most recent 3 releases is semver, and 3 out of the most recent 10 releases is semver, then the project is semver.
```

Comment:
Does `10+` mean 10 and above ? or just above 10?

Reply:
> Does `10+` mean 10 and above ? or just above 10?

I don't know the answer to that. This was just a copy/paste from another part of the docs. @wedamija, @sl0thentr0py - do either of you know what the answer to that?

Reply:
The rules are
 - <= 2
 - > 2 < 10
 - >= 10

Not sure what the best way to word it is

Reply:
Okay so: 
```suggestion
- If 10 or more releases: if any of the most recent 3 releases is semver, and 3 out of the most recent 10 releases is semver, then the project is semver.
```